conventions/documentation/quick-starts.md

```diff
@@ -23,6 +23,7 @@ See the links below to view the latest quick start designs. New designs will be
 * [Catalog page](http://openshift.github.io/openshift-origin-design/designs/developer/4.7/quick-starts-catalog/)
 
 **Quick starts in 4.8**
+* [Card actions](http://openshift.github.io/openshift-origin-design/designs/administrator/4.8/quick-start-actions/)
 * [Code snippets](http://openshift.github.io/openshift-origin-design/designs/administrator/4.8/code-snippets-quickstarts/)
```

Comment:
The path for the code snippets link should be http://openshift.github.io/openshift-origin-design/designs/developer/4.8/code-snippets-quickstarts/